openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
 import moment from 'moment';
 import PropTypes from 'prop-types';
-import { Button, DataTable } from '@edx/paragon';
+import { Button, DataTable, Form } from '@edx/paragon';
+import { Search } from '@edx/paragon/icons';
 
 const getReadableTime = (timestamp) => moment(timestamp).fromNow(true);
 
-const WaitingStepList = ({ studentList, refreshData }) => {
-  const studentListWithTimeAgo = studentList.map((item) => ({
+const currentDate = new Date();
+
+// Calculate the date seven days ago
+const sevenDaysAgo = new Date(currentDate);
+sevenDaysAgo.setDate(currentDate.getDate() - 7);
+
+const myObject = {
+  username: 'sofie_20',
+  graded: 1,
+  graded_by: 1,
+  created_at: sevenDaysAgo,
+  staff_grade_status: 'waiting',
+  workflow_status: '',
+};
+
```

Comment:
what do we need to do to use existing users? This is pretty handy for testing though, at least for now.